metricbeat/metricbeat.full.yml

```diff
@@ -275,6 +275,13 @@ metricbeat.modules:
   period: 10s
   hosts: ["kube-state-metrics:8080"]
 
+# Kubernetes events
+- module: kubernetes
+  metricsets:
+    - event
+  kube_config: ${HOME}/.kube/config
```

Comment:
as per @exekias we can default `in_cluster` to true. this metricset doesnt rely on hosts by default. we can add documentation on various ways to configure it similar to the kubernetes processor.